ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,190 @@
+                                                    name="./buttonText">
+                                            </buttonText>
+                                            <fileInputDragDropText
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="cq/gui/components/authoring/dialog/richtext"
+                                                    fieldLabel="File Input Drag Drop Text"
+                                                    fieldDescription="File Input Drag Drop Description"
+                                                    name="./fileInputDragDropText"
+                                                    useFixedInlineToolbar="{Boolean}true"/>
+                                            <type jcr:primaryType="nt:unstructured"
+                                                  wrapperClass="cmp-adaptiveform-fileinput__type"
+                                                  granite:class="cmp-adaptiveform-fileinput__type"
+                                                  sling:resourceType="granite/ui/components/coral/foundation/form/hidden"
+                                                  fieldLabel="Type of the field"
+                                                  name="./type"
+                                                  value="file">
+                                            </type>
+                                            <showComment
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:hideResource="{Boolean}true">
+                                            </showComment>
+                                        </items>
+                                    </column>
+                                </items>
+                            </columns>
+                        </items>
+                    </basic>
+                    <validation
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <columns
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                    margin="{Boolean}true">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <minimum
```

Comment:
Please re-use all the properties from parent resource type. This should not be copied

Reply:
same discussed this with Sakshi and we concluded the discussion with making V2 as totally separate with no dependency on V1.

Reply:
But why can't we extend from v1? These are just authoring dialogs and I think most of the properties are repeating that of v1

Reply:
Reused the parent resource type by extended from V1 only.